lldb/tools/lldb-dap/README.md

```diff
@@ -27,58 +31,43 @@ get a full featured debugger with a well defined protocol.
 
 # Installation for Visual Studio Code
 
-Installing the plug-in involves creating a directory in any location outside of
-`~/.vscode/extensions`. For example, `~/vscode-lldb` is a valid one. You'll also
-need a subfolder `bin`, e.g. `~/vscode-lldb/bin`. Then copy the `package.json`
-file that is in the same directory as this documentation into it, and symlink
-the `lldb-dap` binary into the `bin` directory inside the plug-in directory.
+Installing the plug-in is very straightforward and involves just a few steps:
 
-Finally, on VS Code, execute the command
-`Developer: Install Extension from Location` and pick the folder you just
-created, which would be `~/vscode-lldb` following the example above.
-
-If you want to make a stand alone plug-in that you can send to others on UNIX
-systems:
+- Install a modern version of node (e.g. `v20.0.0`).
+- On VS Code, execute the command `Install 'code' command in PATH`. You need to
+  do it only once. This enables the command `code` in the PATH.
+- In the `lldb-dap` folder, package and install the extension:
```

Comment:
Maybe we can checking a python script that just does this instead of talking about all the things we need to do? To make this easier, we could make sure a directory could contain all of the stuff in the right structure so we would just copy a folder like ` lldb/tools/lldb-dap/extension` over into the `~/.vscode/extensions/<name>` with a rename. This might be a good place for all of the typescript to be checked in instead of the `src-ts` directory?

Reply:
Tbh these are the minimal amount of steps and they can't really be automated much more than what they are right now. An important detail is that vscode doesn't allow writing directly to the `~/.vscode/extensions` folder anymore for security reasons, so we instead rely on installing the extension through the standard way, which is using a vsix package.

- The first step, which is installing node, depends on your system and your package manager of choice.
- The second step, which is installing the `code` tool, has to be done manually, but only once. 
- The third step, which is packaging and building the extension is trivially two commands that can also be found in the package.json file. `npm run package` and `npm run install`.